app/javascript/src/components/Hero.jsx

```diff
@@ -2,21 +2,21 @@ import React from "react";
 
 import { Button } from "neetoui/v2";
 
-function Hero() {
-  const features = [
-    "Uses Rails, React, Tailwind CSS and Webpacker.",
-    "Uses Devise, Honeybadger, Sidekiq, PostgreSQL, ActiveAdmin.",
-    "Heroku ready. Push to Heroku and it will work.",
-    "Uses slim for cleaner syntax over erb and better performance over haml.",
-    "Intercepts all outgoing emails in non production environment using gem mail_interceptor.",
-    "Uses SemaphoreCI for continuous testing.",
-    "Content compression via Rack::Deflater.",
-    "Auto-formats Ruby code with rubocop.",
-    "Auto-formats JavaScript and CSS code with prettier.",
-    "Performs background job processing 'inline' for heroku env. It means heroku can deliver emails.",
-    "Letter opener gem for development.",
-  ];
+import { FEATURES_LIST } from "./constants";
```

Comment:
LIST is already plural:
```suggestion
import { FEATURE_LIST } from "./constants";
```